src/components/HomepageStyles.scss

```diff
@@ -1,14 +1,135 @@
+.row {
+  padding-left:15px;
+  padding-right:0;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+a {
+  cursor:pointer;
+}
+
+.b {
+  color: black;
+  text-decoration: none;
+}
+
+.w {
+  color: #FFFEE9;
+  text-decoration: none;
+}
+
+.padding{
+  padding-top: 20px;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.NHGD {
+  font-family: neue-haas-grotesk-display, sans-serif;
+  font-weight: 500;
+  font-size: 36px;
+  // content: 'hello';
+}
+
+.NHGT {
```

Comment:
hmmm I'm open to this as a class but I wonder if font should be more of a sectional attribute instead? Especially because will all things with this font have font-weight:400 and font-size:19px?